scrunch/tests/test_utilities.py

```diff
@@ -119,7 +127,7 @@ def test_get_dataset_from_project_no_name(self, session):
         shoji_entity = {
             'element': 'shoji:entity',
             'body': {
-                'id': '123456',
+                'id': 'b2c4c6b7d3a94e58937b23c1fed1b65e',
```

Comment:
Factor this long string into a variable to make it more evident in the test that you're using the exact same string here and in the other places.